docs/usage/ObjectStorage.md

```diff
@@ -204,5 +223,270 @@ aws s3api create-bucket \
      }
     ```
 
+---
+
+## GCP Object Storage
+
+This section covers setup of Object Storage with [*GCP Storage Buckets*](https://cloud.google.com/storage/docs/creating-buckets).
+
+### Create GCP Storage Bucket
+
+```bash
+# Set name of GCP Bucket
+BUCKET=<BUCKET_NAME_HERE>
+
+# Use the gsutil CLI to create the Storage Bucket
+gsutil mb gs://$BUCKET/
+```
+
+### Create a GCP Service Account for Velero to assume
+
+```bash
+# Set PROJECT_ID var to the currently active project
+PROJECT_ID=$(gcloud config get-value project)
+
+# Create a GCP Service Account
+gcloud iam service-accounts create velero \
+    --display-name "Velero Storage"
+
+# Set SERVICE_ACCOUNT_EMAIL to the email associated with the new Service Account
+SERVICE_ACCOUNT_EMAIL=$(gcloud iam service-accounts list \
+  --filter="displayName:Velero Storage" \
+  --format 'value(email)')
+
+
+# Grant necessary permissions to the new Service Account with the gsutil CLI
+ROLE_PERMISSIONS=(
```

Comment:
The UI uses the same SA for bucket access, are there any additional permissions that have to get added for this?

Reply:
Not sure, I just copied these over from Velero docs. Will take a closer look.